swingset/src/main/java/com/nqadmin/swingset/SSBaseComboBox.java

```diff
@@ -254,6 +311,18 @@ protected BaseGlazedModel() {
 	 */
 	protected final SSBaseComboBoxListener ssBaseComboBoxListener = new SSBaseComboBoxListener();
 
+	/**
+	 * List item just used only to workaround GlazedList CONTAINS bug.
+	 */
+	// **** GL STRICT/CONTAINS ****
+	private SSListItem glGlitchItem = null;
+	
+	/**
+	 * Indicates that a garbage string has been entered and the Action Listener should not 
+	 */
+	// **** GL STRICT/CONTAINS ****
```

Comment:
As you can see, I'm trying to flag the workarounds with '**** GL STRICT/CONTAINS ****'